src/backend/db/products.js

```diff
@@ -6,25 +6,156 @@ import { v4 as uuid } from "uuid";
  * */
 
 export const products = [
+  {
+
+    // price range: 100 to 2000, step of 500
+    _id: uuid(),
+    title: "Casual Blue Shirt", 
+    price: 599,
+    rating: "3.7",
+    categoryName: "Men",
+    inStock: true,
+    fastDelivery: true,
+    categoryImg: "https://media.istockphoto.com/photos/latin-man-standing-picture-id825083570?b=1&k=20&m=825083570&s=170667a&w=0&h=Pt9O2-RyYt84bumpiuy3KDUvaDIm--UYutUUnwBKVZM="
```

Comment:
technically its product image not category image

Reply:
yeah, makes sense. Thanks for pointing it out.